proto/viam/app/v1/billing.proto

```diff
@@ -71,6 +71,8 @@ message GetCurrentMonthUsageResponse {
   double discount_amount = 8;
   double total_usage_with_discount = 9;
   double total_usage_without_discount = 10;
+  double monthly_machines_usage_cost = 11;
+  double cost_per_machine_per_month = 12;
```

Comment:
This field seems unnecessary for the same reason we don't have it for other resources.

Reply:
chatted offline, removing and will omit from FE design as well. users can look at invoice for the sku information like this.